distribution/docs/src/main/resources/content/_integrating/_endpoints/nsili-endpoint.adoc

```diff
@@ -1,33 +1,12 @@
 :title: NSILI Endpoint
 :type: endpoint
 :status: published
+:operations: query
 :link: _nsili_endpoint
-:summary: 
-:implements: 
+:url: \${secure_url}/services/nsili.
+:summary: enables a client to search by STANAG 4559 compliant metadata.
+:implements: http://www.gwg.nga.mil/documents/ntb/STANAG_4559_ed2.pdf[NSILI] {external-link}
 
-The NSILI Endpoint enables a client to search by http://www.gwg.nga.mil/documents/ntb/STANAG_4559_ed2.pdf[NSILI] {external-link} compliant metadata.
-
-===== Installing NSILI Endpoint
-
-Install the NSILI Endpoint through the ${admin-console}.
-
-. Navigate to the ${admin-console}.
-. Select the *System* tab.
-. Select the *Features* tab.
-. Install the *catalog-nsili-endpoint* feature.
-
-===== Configuring NSILI Endpoint
+== {title}
 
-Configure the NSILI Endpoint through the ${admin-console}.
-
-. Navigate to the ${admin-console}.
-. Select the *NSILI* application.
-.. If the application is not installed, see <<{reference-prefix}installing_nsili,Installing NSILI>> to install.
-. Select *Configuration*.
-. Select *NSILI Endpoint Configuration*.
-
-See <<{reference-prefix}org.codice.alliance.nsili.endpoint,NSILI Endpoint configurations>> for all possible configurations.
-
-===== NSILI Endpoint URL
-
-The NSILI Endpoint is accessible from \${secure_url}/services/nsili.
+The NSILI Endpoint enables a client to search by http://www.gwg.nga.mil/documents/ntb/STANAG_4559_ed2.pdf[NSILI] {external-link} compliant metadata.
```

Comment:
❓ I assumed we're going to fill this section out quite a bit right?

Reply:
Yes.